src/components/Trade/hooks/useSwapper/useSwapper.tsx

```diff
@@ -54,18 +53,23 @@ export const useSwapper = () => {
   const buyAsset = buyTradeAsset?.asset
   const buyAssetId = buyAsset?.assetId
   const sellAssetId = sellAsset?.assetId
-
-  // Hooks
-  const [swapperManager, setSwapperManager] = useState<SwapperManager>(() => new SwapperManager())
-  const [bestTradeSwapper, setBestTradeSwapper] = useState<Swapper<KnownChainIds>>()
-  const {
-    state: { wallet },
-  } = useWallet()
-  const [receiveAddress, setReceiveAddress] = useState<string | null>()
+  const sellAssetChainId = sellAsset?.chainId
 
   // Selectors
   const flags = useSelector(selectFeatureFlags)
   const assetIds = useSelector(selectAssetIds)
+  const defaultFeeAsset = useAppSelector(state =>
+    selectFeeAssetByChainId(state, sellAssetChainId ?? ''),
```

Comment:
Not sure if the terminology of the const vs. the selector one was on purpose, but it looks like the intent is selecting the *actual* fee asset for that ChainId. Which also works with the `default` terminology, so happy to batch LSP rename `selectFeeAssetByChainId` in a follow-up not to clutter this diff if we prefer to go this route

```suggestion
  const feeAsset = useAppSelector(state =>
    selectFeeAssetByChainId(state, sellAssetChainId ?? ''),
```

or

```suggestion
  const defaultFeeAsset = useAppSelector(state =>
    selectDefaultFeeAssetByChainId(state, sellAssetChainId ?? ''),
```

Reply:
i wouldn't rename the selector, but could get behind renaming variable it's assigned to

Reply:
Good catch, this should really just be `feeAsset`. Renamed in https://github.com/shapeshift/web/pull/3549/commits/1c4205fb3281f4366a66886084b872ad06faa3a7.